add .wled and .log extension

lost-hope opened the pull request:
Add those two extensions, cause they are used by the ARTIF-FX usermod in MM.

lost-hope committed:
add .wled and .log extension

Add those two extensions, cause they are used by the ARTIF-FX usermod in MM.

lost-hope committed:
Update SPIFFSEditor.cpp

Update the gzipped htm array with the two extensions added

softhack007 commented:
👍 +1 as this would allow WLEDMM to also switch to asyncWebserver 2.2.1

willmmiles commented:
The compressed data size is over 1k larger than the previous commit - this seems strange to me for what seems like a very small change to the htm source.  I'm taking a look to see if I can find out why.

willmmiles commented:
> The compressed data size is over 1k larger than the previous commit - this seems strange to me for what seems like a very small change to the htm source. I'm taking a look to see if I can find out why.

Ah, the HTML was minified first.

willmmiles committed:
SPIFFSEditor: Minify HTML

Minify edit.htm with https://kangax.github.io/html-minifier/ before
compressing and uploading.  Saves 1kb.
